projects/v2/src/app/store/sheet.state.ts

```diff
@@ -339,6 +339,17 @@ export class SheetState {
         },
         (error) => {
           console.log(error);
+          const err: Error = {
+            msg: `${error.name} (Status: ${error.status})`,
+            status: error.status,
+            hasError: true,
+            gidErr: sheet.gid ? false : true
```

Comment:
What about when `sheet.gid === 0`?

Reply:
It will catch an error only when gid is not present, that means gid is not provided(Undefined)

Reply:
What are the possible values of gid at that line of code?

Reply:
Got it. I will change for gid ===0.

Reply:
I would do `gidErr: sheet.gid || sheet.gid === 0`. Probably a more 'cool' way to do it, but it gets the job done. You may also rename to `hasGidError` .. `gidErr` feels to me like it might have a value other than true/false in it.